website/api/models/HistoricalUsageSnapshot.js

```diff
@@ -26,6 +26,7 @@ module.exports = {
     hostStatusWebhookEnabled: { required: true, type: 'boolean' },
     numWeeklyActiveUsers: { required: true, type: 'number' },
     hostsEnrolledByOperatingSystem: { required: true, type: 'json' },
+    storedErrors: { required: true, type: 'json' },
```

Comment:
Not sure if this is strictly required, but when I added new usage stats fields, I also added it to `website/assets/js/cloud.setup.js`, you should probably add `storedErrors` there too.

Reply:
thanks! I completely forgot about this after making a mental note when you did this in your PR 🤦 

Added the docs section, and regenerated `cloud.setup.js` via `sails run rebuild-cloud-sdk`.

Reply:
TIL about `sails run rebuild-cloud-sdk` :D I updated it manually, didn't notice that this was generated.